### DISCUSSION TITLE
Dark mode theme color not supported

### DISCUSSION THREAD

**Discussion by jsmit99** [laravel/nova-issues#5707] [Q&A]
- Laravel Version: 10.14.1
- Nova Version: 4.26.4
- PHP Version: 8.1.20
- Database Driver & Version:
- Operating System and Version: iOS
- Browser type and version: Safari (iPhone 13 Pro)

### Description:
The theme-color is always white `#fff` for all nova themes. In dark mode it gives an ugly (white) bar instead of a nice dark bg.

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Go to your local nova installation
2. Log into your account
3. Set dark mode in top right

**Comment by crynobone** [comment]
Unable to reproduce the issue, please provide full reproducing repository based on fresh installation as suggested in the bug report template (or you can refer to https://github.com/nova-issues for example)

**Comment by jsmit99** [comment]
It's about the theme-color (meta setting) in the HTML head section of nova. See attached image.

<img width="283" alt="image" src="https://github.com/laravel/nova-issues/assets/82087876/e0b74f53-4317-466b-9712-70c1066566bd">

For example for Envoyer it will go red
<img width="525" alt="image" src="https://github.com/laravel/nova-issues/assets/82087876/bb9e063c-b987-404b-a294-b6dbca4709da">

**Comment by crynobone** [comment]
You can customise this by creating a partial view file under `/resources/views/vendor/nova/partials/meta.blade.php`

**Comment by jsmit99** [comment]
But shouldnt this be a default setting? The light version just has a basic white background, but the dark version is missing a matching bg.

**Reply by crynobone** [reply]
Some might also want to use the application brand color (just like the example you provide before you removed it)
